test/unit.js

```diff
@@ -28,6 +28,42 @@ metatests.test('Expression chain', async (test) => {
   test.end();
 });
 
+metatests.test('Expression chain with same cells', async (test) => {
+  const sheet = new Sheet();
+  sheet.cells['A1'] = 100;
+  sheet.cells['B1'] = 2;
+  sheet.cells['C1'] = '=A1*B1';
+  sheet.cells['D1'] = '=C1+8+B1';
+  sheet.cells['E1'] = '=D1/2+A1';
+  test.strictSame(sheet.values['D1'], 210);
+  test.strictSame(sheet.values['E1'], 205);
+  test.end();
+});
+
```

Comment:
We have multiple positive tests above, do not need another one

```suggestion
```

Reply:
Ok